pkg/driver/controller_server.go

```diff
@@ -694,6 +694,15 @@ func (driver *Driver) ControllerPublishVolume(ctx context.Context, request *csi.
 		return nil, status.Error(codes.InvalidArgument, "Invalid volume capability specified for ControllerPublishVolume")
 	}
 
+	roVal, err := strconv.ParseBool(request.VolumeContext[readOnlyKey])
```

Comment:
nit: we can rename it to `readOnlyValue` just like the key

Reply:
upadated